src/content/datachannel/filetransfer/js/main.js

```diff
@@ -0,0 +1,171 @@
+/*
+ *  Copyright (c) 2014 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+'use strict';
+
+var localConnection, remoteConnection, sendChannel, receiveChannel,
+  pcConstraint;
```

Comment:
We've just started using JSCS validation with `"preset": "google"`. This adds a few validation rules such as only allowing one var declaration per line.

Could you add this and make the changes? There are less than 10, and they're all pretty trivial. 

FWIW there's a JSCS plugin for Sublime Text, if that helps.